.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - gem --version
   - bundle -v
 script:
+  - if [ -z "$TRAVIS_TAG" ]; then echo "Pull request not labeled."; exit 1; fi
```

Comment:
since `.travis.yaml` is managed my pdk, next time you go to update pdk, these changes are going to be lost :( you have to add them to `.sync.yaml`, then run `pdk update` and they'll be applied to the file

just like https://github.com/puppetlabs/puppetlabs-cron_core/blob/e3c7b78e2b27535d81e47dd49314102f135a193a/.sync.yml#L46-L49

Reply:
thanks for the tip! this is just experimental, turns out it does not work and it's a bit tricky to do this check

Reply:
Checks like this can be an adventure :) Good luck!